package.json

```diff
@@ -31,7 +31,7 @@
     "jasmine": "^3.1.0",
     "nightmare": "^3.0.1",
     "node-fetch": "^2.1.2",
-    "require-yml": "^1.3.1",
+    "require-yml": "^1.3.2",
```

Comment:
is there a reason this changed? I'm not adverse to bumping this, just curious if it's required